include/seqan3/search/fm_index/concept.hpp

```diff
@@ -92,7 +92,7 @@ template <typename t>
 SEQAN3_CONCEPT FmIndex = std::Semiregular<t> && requires (t index)
 {
     typename t::size_type;
-    // typename t::cursor_type;
+    typename t::cursor_type;
```

Comment:
what does this change has to do with this PR?

Reply:
In #1042 I commented it out, but it shouldn't be.
I fixed it here by accident.

I guess we can leave this one-liner in the PR?

Reply:
yes but I propose in different commits